estate/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+{
+    'name': 'Real Estate',
+    'description': 'Real Estate Advertising',
+    # 'version': '1.0',
+    'author': 'robal',
```

Comment:
```suggestion
    'author': 'Odoo S.A.',
```
As an Odoo employee, you are creating this module in the name of Odoo